packages/wpcom.js/examples/media-editor/webpack.config.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {
```

Comment:
How about dropping the build files for the examples? We're not going to be shipping them, and I think they're mostly useful as just code samples anyway. Keeping these targets up and running out of the box is going to add maintenance overhead as webpack releases new major versions, for example.

Reply:
That would mean that we provide some example code that you can study, but you need to BYOBS (bring your own build script) if you want to actually run it? Sounds good to me.

Reply:
Yes, sorry I wasn't very clear, that's exactly what I meant 🙂 And I love the acronym! 😄